### PR TITLE
[RFE] Updating HostCollections and HostContent to be SCA Only

### PR DESCRIPTION
**Pull request by ColeHiggins2**
### Problem Statement
Updating Host-Content and Host Collections to be sca only.

### Related Issues

https://github.com/SatelliteQE/airgun/pull/1425

### CODE REVIEW

**Comment by ColeHiggins2** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contenthost.py -k test_positive_end_to_end
airgun: 1425

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7377
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_contenthost.py -k test_positive_end_to_end --external-logging
Test Result : ==== 1 failed, 1 passed, 21 deselected, 142 warnings in 1588.99s (0:26:28) =====
```

File: tests/foreman/ui/test_hostcollection.py [hunk 1]
```diff
@@ -26,14 +26,14 @@
...
-    with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
+    with Manifester(manifest_category=settings.manifest.golden_ticket) as manifest:
         yield manifest
 
 
 @pytest.fixture(scope='module')
 def module_org_with_parameter(module_target_sat, module_manifest):
     # adding remote_execution_connect_by_ip=Yes at org level
-    org = module_target_sat.api.Organization(simple_content_access=False).create()
+    org = module_target_sat.api.Organization(simple_content_access=True).create()
```
Comment (vsedmik) [inline tests/foreman/ui/test_hostcollection.py:36]:
```suggestion
    org = module_target_sat.api.Organization().create()
```
SCA should be default for new orgs.

**Review by vsedmik — APPROVED** [review]
Codewisely looks good to me, just one minor thing. 
PRT needs a fix.

File: tests/foreman/ui/test_hostcollection.py (hunk 1 above)
Comment (vsedmik) [reply tests/foreman/ui/test_hostcollection.py:36]: We should add BlockedBy to docstring since there is bug reported
```
:BlockedBy: SAT-25817
```

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contenthost.py::test_positive_end_to_end

**Comment by damoore044** [comment]
pytest tests/foreman/ui/test_contenthost.py::test_positive_end_to_end[rhel7_contenthost0-module_repos_collection_with_manifest0]

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contenthost.py::test_positive_end_to_end[rhel7_contenthost0-module_repos_collection_with_manifest0]

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contenthost.py -k 'test_positive_end_to_end and not test_positive_end_to_end_bulk_update'

**Comment by damoore044** [comment]
I just realized `:BlockedBy:` will prevent collection in PRT, disregard those comments lol

**Review by vsedmik — APPROVED** [review]

**Pull request merged**